### ISSUE TITLE
Reaction Count Bug

### ISSUE THREAD

**Issue by annihilatorrrr** [PyrogramMod/PyrogramMod#25]
### Checklist

- [X] I am sure the error is coming from Pyrogram's code and not elsewhere
- [X] I have searched in the issue tracker for similar bug reports, including closed ones
- [X] I ran `pip3 install -U https://github.com/pyrogram/pyrogram/archive/master.zip` and reproduced the issue using the latest development version

### Description

Na

### Steps to reproduce

Just run.

### Logs

```shell
Logs
clz04o021000irqa4t33iuhrv
2024-08-14T16:35:20.784631157Z pyrogram.dispatcher - ERROR - 'NoneType' object has no attribute 'count'

2024-08-14T16:35:20.784663441Z Traceback (most recent call last):

2024-08-14T16:35:20.784669488Z File "/opt/venv/lib/python3.12/site-packages/pyrogram/dispatcher.py", line 296, in handler_worker

2024-08-14T16:35:20.784675555Z await parser(update, users, chats)

2024-08-14T16:35:20.784680787Z File "/opt/venv/lib/python3.12/site-packages/pyrogram/dispatcher.py", line 76, in edited_message_parser

2024-08-14T16:35:20.784686568Z parsed, _ = await message_parser(update, users, chats)

2024-08-14T16:35:20.784692232Z ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

2024-08-14T16:35:20.784697413Z File "/opt/venv/lib/python3.12/site-packages/pyrogram/dispatcher.py", line 69, in message_parser

2024-08-14T16:35:20.784702445Z await pyrogram.types.Message._parse(self.client, update.message, users, chats,

2024-08-14T16:35:20.784707409Z File "/opt/venv/lib/python3.12/site-packages/pyrogram/types/messages_and_media/message.py", line 779, in _parse

2024-08-14T16:35:20.784712500Z reactions = types.MessageReactions._parse(client, message.reactions)

2024-08-14T16:35:20.784717848Z ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

2024-08-14T16:35:20.784723066Z File "/opt/venv/lib/python3.12/site-packages/pyrogram/types/messages_and_media/message_reactions.py", line 54, in _parse

2024-08-14T16:35:20.784728276Z reactions=[types.Reaction._parse_count(client, reaction)

2024-08-14T16:35:20.784733796Z ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

2024-08-14T16:35:20.784737510Z File "/opt/venv/lib/python3.12/site-packages/pyrogram/types/messages_and_media/reaction.py", line 83, in _parse_count

2024-08-14T16:35:20.784741460Z reaction.count = reaction_count.count
```

**Comment by null-nick** [comment]
give me a sample code

**Comment by annihilatorrrr** [comment]
got it automatically on the logs, prolly someone edited a msg in the channel? removed his reaction ig.

**Comment by null-nick** [comment]
fixed 96c8c11af4918fb64a4d87238a3b47931eb1df2f
